appSyncClient.go

```diff
@@ -164,6 +275,36 @@ func (client *appSyncClient) getResolver(params *appsync.GetResolverInput) (*app
 
 }
 
+func (client *appSyncClient) listAllFunctions(apiID string) ([]*appsync.FunctionConfiguration, error) {
```

Comment:
Should we call this `getFunctions` to match the naming convention in this package